connector/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -217,6 +218,28 @@ message Expression {
     bool is_user_defined_function = 4;
   }
 
+  message ScalarInlineUserDefinedFunction {
+    // (Required) Name of the user-defined function.
+    string function_name = 1;
+    // (Required) Indicate if the user-defined function is deterministic.
+    bool deterministic = 2;
+    // (Optional) Function arguments. Empty arguments are allowed.
+    repeated Expression arguments = 3;
+    // (Required) Indicate the function type of the user-defined function.
+    oneof function {
+      PythonUDF python_udf = 5;
```

Comment:
No this is a bug. It's fine to make it id 4.

Reply:
Fixed.